roadmap/propagate-rust-panic-through-native-frame.md

```diff
@@ -0,0 +1,48 @@
+# Propagate Rust panic through native frames
+
+## Summary
+
+These roadmap items refer to when a panic is raised in a Rust function
+that is defined with `"C unwind"` ABI, as would happen if C or C++
+code invoked the following function:
+
+```rust
+extern "C unwind" fn example() {
+    panic!("Uh oh");
+}
+```
+
+In this case, the stack would look like one of the two scenarios:
+
+```
+     ...
```

Comment:
I think these two stack-frame diagrams are not really complete without showing that, to be well defined, the unwind must either be caught in the same language that threw it, or the application must terminate. Or, at least, I believe those are the only cases with which this working group should be concerned.

Reply:
Seems good.